src/pages/index.js

```diff
@@ -131,13 +245,23 @@ function getCardElement(data) {
     imgElement.alt = data.name;
     imgElement.src = data.link;
 
+    if (data.isLiked) {
+        cardLikeBtn.classList.add("card__like-btn_liked");
+    }
+
+
     cardLikeBtn.addEventListener("click", () => {
-        cardLikeBtn.classList.toggle("card__like-btn_liked");
+        if (cardLikeBtn.classList.contains("card__like-btn_liked")) {
+            api.toggleLike(data._id, true)
+                .then(cardLikeBtn.classList.toggle("card__like-btn_liked"));
```

Comment:
`                .then(cardLikeBtn.classList.toggle("card__like-btn_liked"));
`

In `then`, `catch`, `finally` you need to pass only the **reference** to the function (not a function call) otherwise the function is called immediately without waiting for the response from the server. You can pass an **unnamed** arrow function as well. 


```js
        .then(() => {
          cardLikeBtn.classList.toggle("card__like-btn_liked");
        })
        .catch(console.error);  // don't forget about catching errors

```